### ISSUE TITLE
Network images load extremely slowly on Windows

### ISSUE THREAD

**Issue by dharber** [dotnet/maui#6465]
### Description

I've created an app to display an image that changes periodically (every few seconds) with images from a web server. On Windows, it takes 2-3 seconds from when the Image source is set to a UriImageSource until the image actually displays. During that time, the image is blanked out until it's retrieved and decoded.

### Steps to Reproduce

1. Create a new MAUI app
2. Add a name to the Image element
3. Add a timer to periodically set the Image element to a new UriImageSource
4. Run app, images will take seconds to appear after setting the Image source

### Version with bug

Release Candidate 1 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows 10, SDK version unknown and don't know how to find out

### Did you find any workaround?

No workaround found

**Comment by VincentBu** [comment]
repro on Windows. It takes at least 2 seconds to display the image.

**Comment by drasticactions** [comment]
Did you try creating a WinUI app without MAUI and trying to load the images there? If it's slow with the built-in controls, then MAUI can't improve that.

**Comment by Redth** [comment]
I'm not able to reproduce this myself on main.

I suspect it could take time to load an image if it's a large image and takes a bit to download and process it, but since we're using the platform implementation to do that here, there's not really much we can change anyway.

In my example I use a new unique url to load a 400px x 400px image and it loads very quickly.

https://user-images.githubusercontent.com/271950/165131609-e6b5237f-df0c-4500-8fbe-76a7a0310a38.mp4

**Comment by Redth** [comment]
If you can provide a repro project that would be helpful, as in my video above, it works fine here, so I'm not sure how to proceed without more info/repro.

**Comment by dharber** [comment]
Okay, I have to walk that back a bit. On Windows, the size does appear to be making a difference in that larger images definitely take _noticeably_ longer to load. However, even smaller images, such as the avatar images from github, are still taking a significant fraction of a second to load. Even the larger images load significantly faster on Android than the smaller ones on Windows (when they load at all but that's a separate bug).

**Comment by Redth** [comment]
Can you try a plain old WinAppSDK app and see the results?  I'm guessing that's going to be the experience there too.

**Comment by dharber** [comment]
After a bit of gnashing of teeth to figure out how to create a WinAPP SDK app, I finally managed to do so and the images load virtually instantly. So, whatever the problem is appears to be in MAUI itself.

I'll try to create minimal versions of each app that I'm with some appropriately sized images (not sure how I'm going to achieve that but I'll try) and post them here shortly.

**Comment by dharber** [comment]
Hm, I'm starting to suspect the problem isn't with the images at all. I created a small-ish test app that loads images from tumblr and while there is a noticeable lag between setting the image source and when the image actually appears, it's much shorter than when I point to images on my local server.

So I copied the exact same images from tumblr to my server and updated the urls to point to my server and it takes 2-3 seconds for the image to appear.

On Android, the MAUI app doesn't show any delays with either server (tumblr or my local server).  The WinApp SDK app also had very little delay fetching the image.

I am at a loss as to how to proceed other than continuing to write separate app for each platform in a framework that simply works.
